src/sentry/snuba/metrics/datasource.py

```diff
@@ -546,11 +546,7 @@ def get_series(projects: Sequence[Project], query: QueryDefinition) -> dict:
                         ]
                     snuba_query = snuba_query.set_where(where)
 
-                    # Set the limit of the second query to be the provided limits multiplied by
-                    # the number of the metrics requested in the query in this specific entity
-                    snuba_query = snuba_query.set_limit(
-                        snuba_query.limit.limit * len(snuba_query.select)
-                    )
+                    # The initial query already selected the "page", so reset the offset
```

Comment:
@ahmedetefy I bumped into this at some point... now that every metric is a conditional aggregate (e.g. `sumIf(..., metric_id=123)`, we do not need to multiply the snuba limit by the number of selected metrics, right?